scripts/install_all_third_party_dependencies.py

```diff
@@ -1,6 +1,13 @@
 import subprocess
+import sys
 
 from ts_utils.requirements import get_external_stub_requirements
 
-requirements = get_external_stub_requirements()
-subprocess.check_call(("pip", "install", *[str(requirement) for requirement in requirements]))
+
+def main() -> None:
+    requirements = get_external_stub_requirements()
+    subprocess.check_call(("uv", "pip", "install", *sys.argv[1:], *[str(requirement) for requirement in requirements]))
```

Comment:
By forwarding arguments, we naturally allow non-venv (system installs) by letting the script's user follow uv's own helpful hint:
![image](https://github.com/user-attachments/assets/fadc919c-9556-47dc-ab70-dc2c540a15b9)

Reply:
This is a good explanation. It could be a comment in code.